release_docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,204 @@
+        - `$ git commit -m "Reset RELEASE.txt in preparation for the next release."` 
+        - `$ git push` 
+
+9. Package and Distribute Release (Release Manager)
+    - Select release build from workflow.
+        - Choose the release branch
+        - Change ‘Release version tag’ name to 'release' 
+    - Review the release files in Github
+    - Edit the Github Release and change status to Release
+
+10. Conduct Release Retrospective (Release Manager)
+
+| Task Name     | Task Items    |
+| ------------- | ------------- |
+| 3. Prepare Release Notes (Release Manager) | |
+| | 1. Audit Release Changes (Release Manager) | 
+| | 2. Confirm and Update Known Issues (Release Manager or Software Maintainers) | 
+| | 3. Update Test Configuration Coverage Tables (Software Testers or Release Manager) |
+| | 4. Update Supported Operating Environments (Software Testers or Release Manager) |
+| | 5. Review Formatting (Release Manager) |
+| | 6. Review Auxiliary Release Documents (Release Manager) |
+| | 7. Review and Update Installation Instructions (Build System Engineers) | 
+| 4. Freeze Code (Release Manager) | |
+| | 1. Announce Code Freeze (Release Manager) |  
+| | 2. Defer Outstanding Work (Product Manager) |
+| | 3. Verify Regression Test Success (Test Automation Team) |
+| | 4. Verify Third-Party Software (Release Manager) |
+| 5. Update Interface Version (Release  Manager) | |
+| | 1. Create snapshot of current development (Release Manager) | 
+| | 2. Test snapshot files (Release Managers) |
+| | 3. Review Compatibility Report (Release Managers) | 
+| | 4. Verify Changes (Product Manager or Release Managers) | 
+| | 5. Prepare Release Branch (Release Managers) | 
+| 6. Prepare Release Branch (Release Manager) | |
+| | 1. Create Release Branch (Release Manager) | 
+| | 2. Update Dependencies (Release Manager or CMake Lead) | 
+| | 3. Update Version Numbers (Release Manager) | 
+| | 4. Update Default Configuration Mode (Release Manager) |
+| | 5. Lift Code Freeze (Release Manager) | 
+| 7. Perform Release Testing (Test Automation Team or Release Manager or Project Leads) | |
+| | 1. Prepare for Release Testing (Release Manager) | 
+| | 2. Enable Automated Regression Testing (Test Automation Team) | 
+| | 3. Update .so numbers according to the report (Release Manager) |
+| | 4. Execute release workflow |
+| | 5. Notify HDF-Forum (Release Manager) | 
+| | 6. Notify Clients (Project Leads) | 
+| | 7. Sign Binaries (Test Automation Team) | 
+| | 8. Communicate Test Assignments (Release Manager) | 
+| | 9. Perform Manual Source Testing (Software Testers) | 
+| May use binaries from GH | 10. Perform Binary Testing (Software Testers) | 
+| | 11. Report Status (Software Testers)| 
+| | 12. Resolve Issues (Release Manager) |
+| | 13. Review Test Sufficiency (Release Manager)|
+| 8. Finalize Release Notes (Release Manager) | |
+| | 1. Update Release Notes in **release** branch (Release Manager)| 
```

Comment:
I didn't see this line under Task 8 above, just support branch

Reply:
I moved this up as well.